setup.sh

```diff
@@ -53,20 +71,26 @@ prefix-location.u=https://code.metoffice.gov.uk/svn/roses-u
 prefix-web.u=https://code.metoffice.gov.uk/trac/roses-u/intertrac/source:
 prefix-ws.u=https://code.metoffice.gov.uk/rosie/u" >> rose.conf
 
-export PATH=$HOME/.local/rose/bin:$PATH
```

Comment:
JULES instructions suggest best to append the Rose command after generating the above text file

Reply:
ok!